arci/src/clients/joint_trajectory_clients_container.rs

```diff
@@ -106,3 +106,113 @@ where
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn test_container_new() {
+        use super::*;
+        #[derive(Debug, Clone)]
+        struct Dammy {
+            name: Vec<String>,
+        }
+        #[async_trait]
+        impl JointTrajectoryClient for Dammy {
```

Comment:
Typo Dammy -> Dummy. Let's use spell checker in VSCode.
I'm using [this](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)

```suggestion
        impl JointTrajectoryClient for Dummy {
```